torch/utils/cpp_extension.py

```diff
@@ -1356,7 +1379,12 @@ def _prepare_ldflags(extra_ldflags, with_cuda, verbose):
         if with_cuda:
             extra_ldflags.append('-ltorch_hip' if IS_HIP_EXTENSION else '-ltorch_cuda')
         extra_ldflags.append('-ltorch')
-        extra_ldflags.append('-ltorch_python')
+
+        if is_standalone:
+            extra_ldflags.append(f"-Wl,-rpath,{lib_path}")
```

Comment:
FYI, another way to solve this problem is to just set `LD_LIBRARY_PATH` before invoking these executables. rpath is almost certainly more user friendly as it lets people run the binaries by hand.

Reply:
If we use `LD_LIBRARY_PATH` here, then the API is unified on both Linux and Windows. But alternatively, if we use rpath on Linux, as far as I know, there isn't anything equivalent for the MSVC linker. But we can build a bootstrap program that sets the corresponding PATH. What do you think, @ezyang?

Reply:
Making an entire bootstrap program seems like overcall, but setting PATH when we invoke this stuff seems pretty reasonable.

Reply:
So what is the verdict on what `load` should return? (And should we keep `rpath` in Linux, or require users to use `LD_LIBRARY_PATH` for parity?)